includes/class-doubleclick.php

```diff
@@ -153,20 +153,22 @@ public function enqueue_scripts() {
 
 		$data = array(
 			'network_code' => $this->network_code,
-			'mappings' => $mappings,
+			'mappings' => apply_filters( 'dfw_mappings', $mappings ),
 			'targeting' => $this->targeting(),
 		);
 
 		wp_localize_script( 'jquery.dfw.js', 'dfw', $data );
 		wp_enqueue_script( 'jquery.dfw.js' );
 
-		wp_enqueue_style(
-			'dfp',
-			plugins_url( 'assets/css/dfp.css', dirname( __FILE__ ) ),
-			array(),
-			DoubleClick_For_WordPress::VERSION,
-			'all'
-		);
+		if ( ! empty( get_option( 'dfw_css' ) ) ) {
```

Comment:
Does this have a corresponding settings assoc. with it?

Reply:
@aschweigert Were you planning to add anything else to this pull request, or is it okay to close this?